alignak/objects/service.py

```diff
@@ -1471,13 +1346,23 @@ def apply_implicit_inheritance(self, hosts):
                     if host is not None and hasattr(host, prop):
                         setattr(serv, prop, getattr(host, prop))
 
-    def apply_dependencies(self):
+    def apply_dependencies(self, hosts):
         """Wrapper to loop over services and call Service.fill_daddy_dependency()
 
         :return: None
         """
         for service in self:
-            service.fill_daddy_dependency()
+            if service.host and service.host_dependency_enabled:
+                host = hosts[service.host]
+                service.act_depend_of.append(
+                    (service.host, ['d', 'u', 's', 'f'], 'network_dep', '', True)
```

Comment:
I only adapted this from tps://github.com/Alignak-monitoring/alignak/blob/develop/alignak/objects/service.py#L397-L418

We may think later if this is legit or not